benchmark/buffers/buffer-base64-encode.js

```diff
@@ -1,16 +1,20 @@
 'use strict';
 var common = require('../common.js');
 
-var bench = common.createBenchmark(main, {});
+const bench = common.createBenchmark(main, {
+  N: [64 * 1024 * 1024],
```

Comment:
We might want to give this a better name to avoid confusion. I think other benchmarks use names like `len`, which I would be fine with.